### PR TITLE
Fix the error not working all original message transfer protocol

### PR DESCRIPTION
**Pull request by BECATRUE**
Yet, this PR is not completed. Just related to #137.

The problem is all broadcast-receive examples are not working, because it is only encoded by `broadcast()`, not decoded.
So, each communication occurs an error like below.

img

To fix it, I replaced all direct `emit()` into `broadcast()` and changed the receiving part a little.

### CODE REVIEW

**Comment by BECATRUE** [comment]
First of all, I only modified `numgen`, `dbmgr`, and `logger`.
Please leave you opinion about these three apps!

File: examples/numgen.py [hunk 1]
```diff
@@ -83,52 +82,62 @@ def __init__(self, name: str, table: str = "number", parent: Optional[QObject] =
...
+                self.generatorFrame.dbBox.addItem(name)
+        removingDBs = originalDBs - newDBs
+        if self.generatorFrame.dbBox.currentText() in removingDBs:
+            self.generatorFrame.dbBox.setCurrentText("")
+        for name in removingDBs:
+            self.dbs.pop(name)
+            self.generatorFrame.dbBox.removeItem(self.generatorFrame.dbBox.findText(name))
+
+    def receivedSlot(self, channelName: str, content: Any):
+        """Handles the received broadcast message.
```
Comment (kangz12345) [inline examples/numgen.py:124]: Please use the notation agreed in #44.

File: examples/numgen.py [hunk 2]
```diff
@@ -83,52 +82,62 @@ def __init__(self, name: str, table: str = "number", parent: Optional[QObject] =
...
 
-        This is a slot for received signal.
         It assumes that:
             The new database is always added at the end.
             Changing the order of the databases is not allowed.
 
         Args:
-            channelName: A name of the channel that transfered the signal.
-            msg: An input message to be transferred through the channel.
+            channelName: Channel name that transferred the message.
```
Comment (kangz12345) [inline examples/numgen.py:100]: There is no argument `channelName`!

File: examples/logger.py [hunk 3]
```diff
@@ -91,21 +90,41 @@ def frames(self) -> Tuple[LoggerFrame]:
...
         return (self.loggerFrame,)
 
-    @pyqtSlot(str, str)
-    def addLog(self, channelName: str, msg: str):
+    def addLog(self, content: str):
         """Adds a channel name and log message.
 
         Args:
-            channelName: The name of channel
             msg: Log message
```
Comment (kangz12345) [inline examples/logger.py:97]: Argument name mismatch: `content` vs. `msg`.

**Review by kangz12345 — CHANGES_REQUESTED** [review]
As you saw here, we have to manually check the docstring and comments since our tools (e.g., vscode) cannot.
That's why we should try to write the code clearly enough and use comments only when it is necessary.
It would be hard to find outdated comments in the future 😄
Of course, in this PR, they ARE necessary docstrings though!

File: examples/numgen.py (hunk 1 above)
Comment (BECATRUE) [reply examples/numgen.py:124]: Oh, I forgot it. Thanks for letting me know.

**Comment by BECATRUE** [comment]
I applied all reviews!

**Review by kangz12345 — APPROVED** [review]
LGTM

**Comment by BECATRUE** [comment]
I completely forgot fixing other apps.. I will make another PR.

**Pull request merged**